src/EnergyPlus/HeatBalanceManager.cc

```diff
@@ -9912,8 +9912,29 @@ namespace HeatBalanceManager {
     {
         bool ErrorsFound(false); // Flag for input error condition
         bool DoCTFErrorReport(false);
-        for (auto &construction : state.dataConstruction->Construct) {
-            construction.calculateTransferFunction(state, ErrorsFound, DoCTFErrorReport);
+
+        if (state.dataCache->ctfObjectsInCache) {
```

Comment:
Updated to check whether the CTFs can be loaded. If they cannot, compute them as normal.